stories/victory-label.stories.js

```diff
@@ -415,6 +415,30 @@ export const LineHeight = () => {
           />
         }
       />
+      <VictoryScatter
+        {...defaultScatterProps}
+        labelComponent={
+          <VictoryLabel
+            lineHeight={[2, 1, 3]}
+            text={["测试汉字", "不在正常的 ASCII 范围内", "最后一行"]}
+            backgroundStyle={[{ stroke: "blue", fill: "none" }]}
+          />
+        }
+      />
+      <VictoryScatter
+        {...defaultScatterProps}
+        labelComponent={
+          <VictoryLabel
+            lineHeight={[2, 1, 3]}
+            text={[
+              "اختبار اللغات التي تُقرأ من اليمين إلى اليسار",
+              "مثل العربية",
+              "هناك أكثر من ذلك بكثير",
```

Comment:
Looks like something is going wrong here:
https://snapshots.chromatic.com/snapshots/5b4acf7c54c0490024d5980b-6351bb5c8bc2046ba030855b/capture.png

Reply:
Hmm.. I'll take another look, but it *was* working on my machine when I added these. I'll try and figure out what's wrong there.

Reply:
Yeah, I'm not sure.. It looks just fine locally:
<img width="565" alt="image" src="https://user-images.githubusercontent.com/5589147/197248490-38279b86-095e-4182-8f72-5dfe371855f6.png">

Reply:
In fact, if I visit https://5b4acf7c54c0490024d5980b-cxfpozovry.chromatic.com/?path=/story/victorylabel--line-height it also looks correct. Strange...

Reply:
It looks fine on my phone too. But that image above is from Chromatic's screenshots, so I'm not sure what they're doing differently. If we can't reproduce, then we might just want to accept it, because I'm sure it's better than before.

Reply:
Yeah, I'm not having any luck reproducing. What do we know about the environment that chromatic uses to render?

Reply:
Looks like chromatic was using chrome, so.. I'm not sure what's the deal there.

Reply:
@scottrippey I believe we ought to just go ahead and merge, I've not managed to find a repro.

Reply:
Just as a follow-up to this thread:
I was able to reproduce in Safari, so maybe Chromatic is using WebKit too?
Either way, in Safari, the text-layout issue is not new, and I get the same layout in the `main` branch too.  
So, I think we should merge, and address the text-layout issue separately.

Reply:
Ok, that makes sense. I'm on windows, so I unfortunately don't have access to test in safari.